app/controllers/UIModel.scala

```diff
@@ -100,7 +98,12 @@ object AjaxSubscriptionResponse{
   }
 }
 
-object SubscriptionRedirect extends Enumeration {
-  type SubscriptionRedirectType = Value
-  val MANAGE_PAGE, APPLICATION_CHECK_PAGE, API_SUBSCRIPTIONS_PAGE = Value
+sealed trait SubscriptionRedirect extends EnumEntry
+
+object SubscriptionRedirect extends PlayEnum[SubscriptionRedirect] {
+  val values = findValues
+
+  final case object MANAGE_PAGE               extends SubscriptionRedirect
+  final case object APPLICATION_CHECK_PAGE    extends SubscriptionRedirect
+  final case object API_SUBSCRIPTIONS_PAGE    extends SubscriptionRedirect
 }
```

Comment:
Pull this out into another file for consistency

Reply:
:+1: